feat: add `prepr` cms image provider integration (#823)
nuxt/image: docs/content/5.providers/prepr.md

# Prepr

Nuxt Image integration with Prepr CMS

Integration between [Prepr](https://prepr.io/) and Nuxt Image.

To use this provider you just need to specify the `projectName` of your project in Prepr.

```ts [nuxt.config.ts]
export default defineNuxtConfig({
  image: {
    prepr: {
      // E.g.: https://YourProjectName.prepr.io/
      projectName: 'YourProjectName',
    }
  }
})
```

## Modifiers

The Prepr provider supports a number of additional modifiers. For a full list,
check out [the Prepr documentation](https://docs.prepr.io/reference/rest/v1/assets-resizing/).

> All current transformations currently mentioned in Prepr docs are supported. According
> to Prepr Support team, more modifiers will be added soon, and the documentation
> page is due for an update.

Reviewer: ```suggestion
All current transformations currently mentioned in Prepr docs are supported.
```

For the time being you might find the following links useful:

- [Assets Resizing via REST API](https://docs.prepr.io/reference/rest/v1/assets-resizing/)
- [Understanding your marketing and design team workflows](https://docs.prepr.io/managing-content/images)

::alert{type="info"}

> prepr.io does not provide away to restrict what domains can

Reviewer: oops, missed this:

```suggestion
prepr.io does not provide a way to restrict what domains can
```

> Unfortunatly prepr.io still does not provide away to restrict what domains can

Reviewer: ```suggestion
prepr.io does not provide away to restrict what domains can
```
